sporkfish/lichess_bot/lichess_bot_berserk.py

```diff
@@ -144,13 +145,39 @@ def set_pos_and_play_move(
                 num_moves = len(state["moves"].split())
                 prev_moves = state["moves"]
                 set_pos_and_play_move(num_moves, color, prev_moves, game_id, state)
+            elif state["type"] == "opponentGone" and state["gone"]:
+                time.sleep(8)
+                self.client.board.claim_victory(game_id)
+
+    def should_accept_challenge(self, event):
+        bool_challenge = True
+        if (
+            event["challenge"]["speed"] in {"rapid", "bullet", "blitz"}
+            and event["challenge"]["variant"]["key"] == "standard"
+        ):
+            bool_challenge
+        else:
+            bool_challenge = False
+        return bool_challenge
 
     def run(self) -> None:
         """
         Start the Lichess bot, listening to incoming events sequentially and playing games accordingly.
         """
-
         events = self.client.bots.stream_incoming_events()
         for event in events:
-            if event.get("type") == "gameStart":
+            if event.get("type") == "challenge":
+                if self.should_accept_challenge(event):
+                    self.client.bots.accept_challenge(event["challenge"]["id"])
+                else:
+                    self.client.bots.decline_challenge(event["challenge"]["id"])
+            elif event.get("type") == "gameStart":
                 self._play_game(event["game"]["fullId"])
+            elif event.get("type") == "gameFinish":
+                self.client.bots.post_message(
+                    event["game"]["fullId"],
+                    "GGWP, Hope you had fun playing with Sporkfish!",
```

Comment:
Same here, putting it into a class variable would be better. If people want to change the message they will have to scroll down here to understand that this part is posting the message. Putting it at the start of the class as variable means they can a) change it without understanding the code below and b) find the references that use it, should they need to.